lib/src/commands/create/templates/very_good_core/very_good_core_bundle.dart

```diff
@@ -1626,15 +1626,15 @@ final veryGoodCoreBundle = MasonBundle.fromJson(<String, dynamic>{
     {
       "path": "pubspec.yaml",
       "data":
-          "bmFtZTogdmVyeV9nb29kX2NvcmVfaG9va3MKCmVudmlyb25tZW50OgogIHNkazogIj49Mi4xNy4wIDwzLjAuMCIKCmRlcGVuZGVuY2llczoKICBtYXNvbjogXjAuMS4wLWRldi40MAoKZGV2X2RlcGVuZGVuY2llczoKICB2ZXJ5X2dvb2RfYW5hbHlzaXM6IF4zLjEuMAo=",
+          "bmFtZTogdmVyeV9nb29kX2NvcmVfaG9va3MKCmVudmlyb25tZW50OgogIHNkazogIj49Mi4xNy4wIDwzLjAuMCIKCmRlcGVuZGVuY2llczoKICBtYXNvbjogXjAuMS4wLWRldi40MAoKZGV2X2RlcGVuZGVuY2llczoKICB2ZXJ5X2dvb2RfYW5hbHlzaXM6IF40LjAuMAo=",
       "type": "text"
     }
   ],
   "name": "very_good_core",
   "description": "A Very Good Flutter app created by Very Good Ventures.",
-  "version": "0.1.13",
+  "version": "0.1.14",
   "environment": {"mason": ">=0.1.0-dev.40 <0.1.0"},
-  "repository": "https://github.com/VeryGoodOpenSource/very_good_core",
+  "repository": "https://github.com/ianmaciel/very_good_core",
```

Comment:
Extraneous change

Reply:
I'm sorry, this was'nt supposed to be commited.